nilearn/reporting/data/html/report_body_template.html

```diff
@@ -60,13 +60,28 @@
 */
 
 div.nilearn_report {
-    padding-top: 5px;
+    padding-top: 0px;
+    border: solid rgb(150, 150, 150);
+    border-width: 1pt;
+    border-radius: 6pt;
 }
 
 /* Isolate us a bit from the styles specified by other stylesheets */
 div.nilearn_report h1 {
     text-align: left;
     padding-top: 15px;
+    margin-left: 0pt;
+    margin-right: 0pt;
+    padding-top: 5pt;
```

Comment:
We're defining `padding-top` twice, here. Can we remove this one and modify the previous definition ?
```suggestion
```

Reply:
Good catch!

Fixed in the PR.